utils/rpms/daos.rpmlintrc

```diff
@@ -44,7 +44,10 @@ addFilter("E: static-library-without-debuginfo \/usr\/lib64\/lib(dfuse|ioil)\.a"
 
 # these need to be fixed:
 # https://daosio.atlassian.net/browse/DAOS-11539
-addFilter("W: no-soname \/usr\/lib64\/lib(ds3|daos_(common|cmd_hdlrs|tests|serialize|common_pmem)|dfs|dfuse|duns|ioil|dpar(|_mpi)).so")
+addFilter("W: no-soname \/usr\/lib64\/lib(ds3|daos_(common|cmd_hdlrs|tests|serialize|common_pmem)|dfs|dfuse|duns|ioil|pil4dfs|dpar(|_mpi)).so")
 
 # Tests rpm needs to be able to build daos from source so pulls in build deps and is expected.
 addFilter("daos-client-tests.x86_64: E: devel-dependency protobuf-c-devel")
+
+# libcapstone.so only exists in libcapstone-devel on Leap
+addFilter("daos-client.x86_64: E: devel-dependency libcapstone-devel")
```

Comment:
This doesn't seem correct.  Binaries should only use SOVERSIONed public libraries.  Unversioned libraries such as are found in -devel packages are not really even libraries but are symlinks to the versioned library.  On SUSE platforms those are typically in an RPM with the name formatted as <libname><SOVERSION> such as libfabric1 or libfoobar3, etc.

Reply:
libcapstone.so is ONLY provided in libcapstone-devel on Leap 15.

Reply:
I removed this line. Thank you!